README.md

```diff
@@ -16,20 +20,20 @@ pip install opentelemetry-exporter-dynatrace-metrics
 
 ### Usage
 
-The general setup of OpenTelemetry Python is explained in the official [Getting Started Guide](https://opentelemetry-python.readthedocs.io/en/stable/getting-started.html).
+The general setup of OpenTelemetry Python is explained in the official [Getting Started Guide](https://open-telemetry.github.io/opentelemetry-python/getting-started.html#add-metrics).
 
 ```python
 # configure API endpoint and authentication token
 exporter = DynatraceMetricsExporter(endpoint_url, api_token)
 
 # setup metrics export pipeline
-_metrics.set_meter_provider(MeterProvider(
+metrics.set_meter_provider(MeterProvider(
         metric_readers=[PeriodicExportingMetricReader(
```

Comment:
I think this way it actually sets it up as all cumulative, is that possible?

Reply:
Yes, this was an oversight. Thought it was still working the way it did before, but it does not.
Added temporality preference in [2d01775](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/2d01775f272252daa002b671d1a760238bbcfc8c).